### ISSUE TITLE
Release builds for Scala 2.12

### ISSUE THREAD

**Issue by DJFliX** [justwrote/scala-faker#7]
Hi! I'm currently upgrading my application and one of the dependencies that does not yet have published artifacts for Scala 2.12 is your scala-faker. Could you release a build for 2.12 please?

**Comment by Uxio0** [comment]
+1

**Comment by DJFliX** [comment]
Because apparently this isn't of any priority to the original author, I've published this package to my own repo based on the [fork](https://github.com/Eldius/scala-faker)/[pull request](https://github.com/justwrote/scala-faker/pull/9) by Eldius:  . Hope someone finds it useful.

Repo:
`"qwiksense-misc" at "https://packagecloud.io/priv/e42a90b412b22c1f4da033ec92b8df822c7e137f25274573/qwiksense/misc/maven2", // scala-faker`
Package name:
`"se.qwiksen" %% "scala-faker" % "0.5-QWIKSENSE" % Test`

**Comment by j1mr10rd4n** [comment]
Hi

**Comment by j1mr10rd4n** [comment]
Agreed, it seems the original author doesn't have time to maintain this repo. I found the faker gem very useful in my ruby jobs so I'd like to see it continue to be useful in scala land.

I've merged all the outstanding PRs (#4, #6, #8, #9) to a fork at [bitblitconsulting](https://github.com/bitblitconsulting/scala-faker) and done some general updating to the build to bring it from sbt-0.12 to sbt-0.13 standards.

I've also cut a 0.4 release and published on sonatype/maven central:
```"it.bitbl" %% "scala-faker" % "0.4" % Test```

Ruby's [faker](https://github.com/stympy/faker) has undergone a lot of developent since this repo was last maintained; my intention is to bring the new categories of fakes over here too - contributions and pull requests will be gladly accepted!

**Comment by alvarolorentedev** [comment]
+1
